packages/okta-react/test/jest/auth.test.js

```diff
@@ -158,6 +158,35 @@ describe('Auth configuration', () => {
     }
     expect(createInstance).toThrow();
   });
+
+  it('accepts options in camel case', () => {
+    function createInstance () {
+      return new Auth({
+        issuer: 'https://foo/oauth2/default',
+        clientId: 'foo',
+        redirectUri: 'https://foo/redirect'
+      });
+    }
+    expect(createInstance).not.toThrow();
```

Comment:
Do you mean to CALL createInstance here, or am I missing something?

Reply:
I believe `toThrow` takes the argument originally passed to expect and invokes it within a try / catch block.

Reply:
...how have I not known this?  That's cray.